src/github.com/getlantern/lantern-ui/app/angular-feed/scripts/services.js

```diff
@@ -37,7 +37,8 @@ angular.module('feeds-services', []).factory('feedService', ['$q', '$http', '$sc
           deferred.reject(new Error("invalid data format"));
           return
         }
-        feedCache.set(feedURL, data);
+        // Always get fresh copy, in hope to be more reactive to network errors
+        // feedCache.set(feedURL, data);
```

Comment:
It doesn't happen to me now, could be caused by network glitches. It may take a bit longer to load feeds when Lantern starts up and old config doesn't work any more.

One possible way is to load cached feeds first, fetch feeds in background and replace it when succeeded, or show error page if failed. Not sure if I have a chance to make that change today.